onnxscript/rewriter/pattern.py

```diff
@@ -975,7 +982,7 @@ def _apply_to_graph_or_function(
         for rule in self.rules:
             deltas = []
             for i, node in enumerate(graph_or_function):
-                delta = rule.try_rewrite(model, node)
+                delta = rule.try_rewrite(model, graph_or_function, node)
```

Comment:
I think generic_pattern.py also has an implementation of `try_rewrite` ... at least the signature will have to be changed there?

Reply:
You are right. I don't find generic_pattern.py updates opset_imports. Do you recall where it might be, or you add it somewhere?

Reply:
Right, I don't think the generic pattern matcher is handling the introduction of new opsets. Agree it can be done separately (ideally by using same implementation).

Reply:
I tried but we would get this instead:

```
onnxscript/rewriter/pattern_test.py:360: in test_opset_import_in_function
    onnx.checker.check_model(model_ir)
../anaconda3/envs/main-dev/lib/python3.11/site-packages/onnx/checker.py:179: in check_model
    C.check_model(
E   onnx.onnx_cpp2py_export.checker.ValidationError: No Op registered for afunction with domain_version of 17
```

It lookes like checker does not like the idea of ("", 17)?